### PR TITLE
Nerf psy drain

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Psy drain will now give 40 psy points rather than 60. Amount of larva unchanged

## Why It's Good For The Game

Rebalance psy drain a bit. This is to make cocooning still competitive, while making xeno steamrolling less.

## Changelog
:cl:
balance: Psy drain will now give 40 psy points rather than 60. Amount of larva unchanged
/:cl:

### CODE REVIEW

**Comment by Losenis** [comment]
4 marines dying would give the equivalent of having a pool for around 16 minutes, and it'd then need another 16 to actually see a profit to then have a young xeno where he'd have otherwise been ancient or elder already attacking marines in the meantime. Rounds are already getting close to that timeframe on shutters opening and the game being already decided. It'd make the failure of the first push cement the marine loss even harder if xenos decide to psydrain everyone for 2 or 3 free larva assuming a push of 8~12 to immediately start fobsieging, not taking into account the inevitable stragglers and ungas here and there that would give more larvas as well.

**Comment by BraveMole** [comment]
> 4 marines dying would give the equivalent of having a pool for around 16 minutes, and it'd then need another 16 to actually see a profit to then have a young xeno where he'd have otherwise been ancient or elder already attacking marines in the meantime. Rounds are already getting close to that timeframe on shutters opening and the game being already decided. It'd make the failure of the first push cement the marine loss even harder if xenos decide to psydrain everyone for 2 or 3 free larva assuming a push of 8~12 to immediately start fobsieging, not taking into account the inevitable stragglers and ungas here and there that would give more larvas as well.

I mean, it was like that in old distress (one xenos every 4.5 marine), and xenos were terrible at fobsieging. I'd say that cocoon is much better when you are trying to siege and you destroyed the marine push, and psy drain is better when you are frantikly defending

**Comment by Losenis** [comment]
If xenos are fobsieging they've essentially won the game if they're not bald. What loses them the game there is xenos suiciding at cades in enough numbers that pools won't even catch up properly, because it'll still be just one larva every 16/8/however many minutes. You have 15 xenos fobsieging for example, you'd rather quickly increase that to 16 or 17 rather than just cocoon and then wait to have enough points for another pool, and then wait 16 minutes to break even, and then another 16 to actually make a profit. The only times a round actually lasts that long from a fobsiege is when one side is winning so hard that the round is already decided.

This also just enables pool-less gameplay even harder by letting xenos rampage around without pools and just psydrain marines that go out of cades without direction to just try have fun with a game situation that is essentially "stay in fob cadehugging for an hour doing nothing to force the xenos to make a silo since it's only till 10 mins past the hour that they lose from it". It's not fun gameplay. I haven't seen it in action but with small pools the xenos might even have the chance to make one at the last moment to avoid collapse and have it be expendable, then when it gets destroyed build another elsewhere.

**Comment by BraveMole** [comment]
> If xenos are fobsieging they've essentially won the game if they're not bald. What loses them the game there is xenos suiciding at cades in enough numbers that pools won't even catch up properly, because it'll still be just one larva every 16/8/however many minutes. You have 15 xenos fobsieging for example, you'd rather quickly increase that to 16 or 17 rather than just cocoon and then wait to have enough points for another pool, and then wait 16 minutes to break even, and then another 16 to actually make a profit. The only times a round actually lasts that long from a fobsiege is when one side is winning so hard that the round is already decided.
> 
> This also just enables pool-less gameplay even harder by letting xenos rampage around without pools and just psydrain marines that go out of cades without direction to just try have fun with a game situation that is essentially "stay in fob cadehugging for an hour doing nothing to force the xenos to make a silo since it's only till 10 mins past the hour that they lose from it". It's not fun gameplay. I haven't seen it in action but with small pools the xenos might even have the chance to make one at the last moment to avoid collapse and have it be expendable, then when it gets destroyed build another elsewhere.

There's no reason not to allow pool-less gameplay.

**Comment by Losenis** [comment]
> There's no reason not to allow pool-less gameplay.

It's not fun. If that doesn't count as a reason, stop pretending it's a game that you're coding. Also the last sentence I wrote on the last post ended up happening too. Xenos just built pools to avoid the timer and didn't even care about it getting destroyed, and banked psy points to replace it should it have happened. It was a queen and shrike just running around the map and about to stealthjack, and it was only the two of them rather than multiple with this psydrain PR + call burrowed. They were running around like that for almost an hour.

image

The counter to pool-less gameplay is alt tab in fob and wait till the game forces them to build a pool, assuming you don't just waste your time going towards one single pool that the xenos bank psy points on to replace rather than have it function as an actual pool instead of just bait and bypassing mechanics, so you have to go back to fob and wait again for the game to force them to build another. If you want pool-less gameplay just remove pools altogether, the entire points of pools was to avoid the shitshow that was silos.

**Comment by BraveMole** [comment]
> > There's no reason not to allow pool-less gameplay.
> 
> It's not fun. If that doesn't count as a reason, stop pretending it's a game that you're coding. Also the last sentence I wrote on the last post ended up happening too. Xenos just built pools to avoid the timer and didn't even care about it getting destroyed, and banked psy points to replace it should it have happened. It was a queen and shrike just running around the map and about to stealthjack, and it was only the two of them rather than multiple with this psydrain PR + call burrowed. They were running around like that for almost an hour.
> 
> image
> 
> The counter to pool-less gameplay is alt tab in fob and wait till the game forces them to build a pool, assuming you don't just waste your time going towards one single pool that the xenos bank psy points on to replace rather than have it function as an actual pool instead of just bait and bypassing mechanics, so you have to go back to fob and wait again for the game to force them to build another. If you want pool-less gameplay just remove pools altogether, the entire points of pools was to avoid the shitshow that was silos.

Pools had two objectives:
- giving marines an objective to push
- making cadehugging easier to deal with as a xeno

I can't satisfy everyone. There's a shit ton of people that were complaining about the original design of pool, hence we are here.
This is a compromise, that is trying to keep the good part of old distress while adding the two objectives.

So yes, xenos can play pool less, because players wanted it. That's how it is, period. 

Of course, if we are emulating silo gameplay, we are taking some of the default of this gamemode.
Frankly, don't blame me, blame the players that wanted that.

**Comment by Losenis** [comment]
> blame the players that wanted that

I don't want to start sounding like a certain guy but I haven't seen anyone liking the latest iteration very much other than "well at leat it's not silos". This all just feels like you're getting peer pressured into trying to satisfy both sides and ending up satisfying none because rather than a compromise it's just getting the problems of both in one neat package. People who don't like pools are forced to build and defend/try to tard wrangle the entire marine side to attack pools and people who didn't like silos now will have cadehugging pool-less gameplay on top of that with no objective other than going out to a pool placed to be expendable at 13:09 and rebuilt wherever else once it's down.

First push dying already cements marines losing because xenos will only get stronger, and if you couldn't push out with the entire marine force you're simply never going to stand a chance vs stronger xenos with less marines. With this pr you could have the first push die and then fob be immediately sieged by upwards of 5 or 6 more xenos if all get psydrained, which will very likely be played since all the marines dying would sooner wait 2 mins to play xeno rather than 30 mins to respawn in a round they'll most likely see the end stats of while prepping if it doesn't end sooner.

So far today that I've seen there was only one round that dragged out enough for cocooning to theoretically have an actual effect, which was the pool-less delay round that didn't even count due to no pool other than single expendable ones. Many times they even have leftover psi points because at that point they're not even needed since marines are already killed off or just getting sieged with boilers, or the push continues so you have to get larva now rather than later.

The PR won't have the effect you intent for it to have. Or at least I hope you don't intend for this.

**Comment by BraveMole** [comment]
> People who don't like pools are forced to build and defend/try to tard wrangle the entire marine side to attack pools

No, they don't? Just don't build pool

> With this pr you could have the first push die and then fob be immediately sieged by upwards of 5 or 6 more xenos if all get psydrained, which will very likely be played since all the marines dying would sooner wait 2 mins to play xeno rather than 30 mins to respawn

If all your push died without killing enough xenos, you just deserve to lose. If you did killed xeno in your push, and xeno are psydraining you, the ratio of xeno/marine should roughly stay the same (as a marine spawning give more than a fourth of a xeno).

> So far today that I've seen there was only one round that dragged out enough for cocooning to theoretically have an actual effect

And i don't see how pool changed anything in that regard. Players changed their way of playing, the way xeno respawn is barely relevant to that. In fact, pools are barely relevant in the first 30 minutes of the round

> Many times they even have leftover psi points because at that point they're not even needed since marines are already killed off or just getting sieged with boilers

And i keep telling ungas they should stop bumrush, but they do. Their own fault. Everytime they rush with zero cades and miner behind them, they are asking for a quick defeat against a competent hive. This always was the case, even in old distress

File: code/modules/mob/living/carbon/xenomorph/abilities.dm [hunk 1]
```diff
@@ -1187,10 +1187,8 @@
 	use_state_flags = XACT_USE_STAGGERED|XACT_USE_FORTIFIED|XACT_USE_CRESTED //can't use while staggered, defender fortified or crest down
 	keybind_signal = COMSIG_XENOABILITY_HEADBITE
 	plasma_cost = 100
-	///How much psy points it give
-	var/psy_points_reward = 60
 	///How much larva points it gives (8 points for one larva in distress)
-	var/larva_point_reward = 1
+	var/larva_point_reward = 2
```
Comment (BraveMole) [inline code/modules/mob/living/carbon/xenomorph/abilities.dm:1191]:
```suggestion
	var/larva_point_reward = 1.5
```

**Comment by Losenis** [comment]
I had also forgotten to mention, psydraining marines has the benefit of letting them permadie faster instead of waiting for cocoon that may eventually get dragged and have the marine come out with a reset defib timer.

**Comment by lKiseki** [comment]
Taking into account that this exists alongside #6554, the difference between cocooning and psydraining wouldn't be that big, which means cocooning would lose value since psydraining has the advantage of not freezing the DNR timer. Either cocooning should have a bigger advantage, like taking less time to fully generate points, or psydraining should have a decreased reward so as to maintain cocooning's value.

**Pull request closed without merging**